app/controllers/session_answers_controller.rb

```diff
@@ -2,7 +2,7 @@ class SessionAnswersController < ApplicationController
   before_action :set_cache_headers
 
   def start
-    redirect_to session_flow_path(id: params[:id], node_name: next_node_name)
+    redirect_to session_flow_path(id: params[:id], node_path: next_node_path)
```

Comment:
Not sure if node_path describes this variable well. Were there any other name you were considering?

Reply:
I'm not too happy with the name, but couldn't think of a better alternative. node_path_segment perhaps. Happy to consider something else.

Reply:
I've replaced `node_path` with `node_slug`